packages/Radio/stories/examples/Showcase.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { boolean, select, text } from "@storybook/addon-knobs";
+import { Rule, Tagline } from "storybook/assets/styles/common.styles";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import Heading from "@paprika/heading";
+import { RadioStory } from "../Radio.stories.styles";
+import Radio from "../../src/Radio";
+
+const radioProps = () => ({
+  size: select("size", ShirtSizes.DEFAULT, "medium"),
+  isDisabled: boolean("isDisabled", false),
+  canDeselect: boolean("canDeselect", false),
+  a11yLabelledByText: text("a11yLabelledByText", ""),
+});
+
+const ExampleStory = props => {
+  return (
+    <RadioStory>
+      <Heading level={1} displayLevel={2} isLight>
+        Showcase
+      </Heading>
+      <Tagline>Use the knobs to tinker with the props.</Tagline>
+      <Rule />
+      <Radio.Group
+        defaultCheck={check => check.props.value.id === "value 1"}
+        onChange={value => {
+          console.log(value);
```

Comment:
Recommended: use Storybook `action` here, like this:

https://github.com/acl-services/paprika/blob/master/packages/Button/stories/examples/Basic.js#L19